src/app/app.component.ts

```diff
@@ -537,7 +538,10 @@ export class AppComponent implements OnInit {
   getInterpolationType(curveType) {
     return this.curves[curveType] || this.curves['default'];
   }
-
+  isTensionSliderRequired(curveType) {
+    return this.curves[curveType] == this.curves['Cardinal'] ||
+    this.curves[curveType] == this.curves['Catmull Rom'];
```

Comment:
Indent to match coding style.
```suggestion
    return this.curves[curveType] == this.curves['Cardinal'] ||
      this.curves[curveType] == this.curves['Catmull Rom'];
```